README.md

```diff
@@ -728,79 +772,94 @@ environments should be exposed as an input variable. For example, the frontend-a
 as you may want to run smaller/fewer servers in staging than in prod to save money.
 
 In a separate repo, called, for example, infrastructure-live, you define the code for all of your environments, which
-now consists of just two files per component:
+now consists of just one `.tfvars` file per component (e.g. `frontend-app.tfvars`, `backend-app.tfvars`, etc). This 
+file defines a `terragrunt = { ... }` block to configure Terragrunt, as well as the environment-specific values for the
+input variables for that component. 
 
-1. `terraform.tfvars`: This file defines the environment-specific values for all of your input variables. For example,
-   for `stage/frontend-app/terraform.tfvars`, you might have:
+For example, `stage/frontend-app.tfvars` may look like this:
    
-    ```hcl
-    instance_count = 3
-    instance_type = "t2.micro"
-    ```
+```hcl
+terragrunt = {
+  include {
+    path = "${find_in_parent_folders()}"
+  }
+}
 
-    Whereas for `prod/frontend-app/terraform.tfvars`, you might have:
-   
-    ```hcl
-    instance_count = 10
-    instance_type = "m2.large"
-    ```
-1. `.terragrunt`: This is the typical Terragrunt configuration file, which specifies how to manage remote state and 
-   locking. 
+instance_count = 3
+instance_type = "t2.micro"
+```
+
+And `prod/frontend-app.tfvars` may look like this:
```

Comment:
You're right, the structure of the docs was confusing. It's a leftover from when the `source` stuff was defined in a separate file (`.terragrunt`). I've re-ordered it for clarity here: https://github.com/gruntwork-io/terragrunt/pull/122/commits/56c04585074fe4df78f7e7ceba29416d66fa7854